factorio

```diff
@@ -442,6 +447,34 @@ case "$1" in
       exit 1
     fi
     ;;
+	
+  new-game)
+    # Stop Service
+    if is_running; then
+      stop_service
+    fi
+
+    if ! as_user "$BINARY --create ${SAVE_NAME}"; then
+      echo "Failed to create new game"
+	  exit 1
+    else
+      echo "New game created"
+    fi
+    ;;
+	
+  save-game)
+    savename=${FACTORIO_PATH}/saves/$2.zip
```

Comment:
I recently implemented the ${WRITE_DIR} variable to ensure users can choose where factorio stores its data, please use this one rather than FACTORIO_PATH